src/main/java/frc/robot/commands/TeleopCommand.java

```diff
@@ -3,38 +3,52 @@
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.io.Controls;
 import frc.robot.subsystems.drive.Drive;
+import frc.robot.subsystems.drive.Gyroscope;
 
 public class TeleopCommand extends CommandBase {
 
     private Drive drive;
+    private Gyroscope gyroscope;
     private Controls controls;
     private double DEADZONE = 0.11;
-    private boolean resetEncoders;
+    double desiredAngle;
+    boolean drivingStraight = false;
   
-    public TeleopCommand(Controls controls, Drive drive) {
+    public TeleopCommand(Controls controls, Drive drive, Gyroscope gyroscope) {
```

Comment:
If this drive straight code doesn't work I suggest you revert these changes so you don't overwrite the changes in master.